app-backend/rest-api/handler.js

```diff
@@ -1,36 +1,34 @@
-import 'babel-polyfill';
-import { graphqlLambda, graphiqlLambda } from 'apollo-server-lambda';
-import lambdaPlayground from 'graphql-playground-middleware-lambda';
-import { makeExecutableSchema } from 'graphql-tools';
+import { ApolloServer } from 'apollo-server-lambda';
 import { schema } from './schema';
 import { resolvers } from './resolvers';
 
-const myGraphQLSchema = makeExecutableSchema({
+const server = new ApolloServer({
   typeDefs: schema,
   resolvers,
-  logger: console,
+  formatError: error => {
+    console.log(error);
+    return error;
+  },
+  formatResponse: response => {
+    console.log(response);
+    return response;
+  },
+  context: ({ event, context }) => ({
+    headers: event.headers,
+    functionName: context.functionName,
+    event,
+    context,
+  }),
+  playground: {
+    endpoint: process.env.REACT_APP_GRAPHQL_ENDPOINT
+      ? process.env.REACT_APP_GRAPHQL_ENDPOINT
+      : '/production/graphql',
+  },
+  tracing: true,
 });
 
-exports.graphqlHandler = function graphqlHandler(event, context, callback) {
-  function callbackFilter(error, output) {
-    // eslint-disable-next-line no-param-reassign
-    output.headers['Access-Control-Allow-Origin'] = '*';
-    callback(error, output);
-  }
-
-  const handler = graphqlLambda({ schema: myGraphQLSchema, tracing: true });
-  return handler(event, context, callbackFilter);
-};
-
-// for local endpointURL is /graphql and for prod it is /stage/graphql
-exports.playgroundHandler = lambdaPlayground({
-  endpoint: process.env.REACT_APP_GRAPHQL_ENDPOINT
-    ? process.env.REACT_APP_GRAPHQL_ENDPOINT
-    : '/production/graphql',
-});
-
-exports.graphiqlHandler = graphiqlLambda({
```

Comment:
graphiql is deprecated in Apollo Server 2.0

Reply:
@amerani why did you remove playground integration?

Reply:
I did not! It's enabled in the options object passed to `ApolloServer` constructor. see line 22